packages/build/tests/error/fixtures/dependencies/netlify.toml

```diff
@@ -1,2 +1,2 @@
-[[plugins]]
-package = "netlify-plugin-test-does-not-exist"
+[build]
+functions = "functions"
```

Comment:
This test is checking how errors of type `dependencies` (i.e. `npm install` errors) are reported to Bugsnag.
It must be switched to another error of that type since this PR makes it harder to emulate plugin installation errors.